ccam/Makefile

```
@@ -0,0 +1,4 @@
+main:
+	gcc ucam.c -o ucam
+clean:
```

clean should be a phony rule.
```suggestion
.PHONY: clean
clean:
```